sys/backend_proxy.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const P = require('bluebird');
+const HyperSwitch = require('hyperswitch');
+
+const Template = HyperSwitch.Template;
+const HTTPError = HyperSwitch.HTTPError;
+
+module.exports = (options) => {
+    options = options || {};
+    options.backend_host_template = options.backend_host_template || '/{domain}/sys';
+    if (!Object.prototype.hasOwnProperty.call(options, 'use_path_segment')) {
+        options.use_path_segment = true;
+    }
+    if (!Object.prototype.hasOwnProperty.call(options, 'block_external_reqs')) {
+        options.block_external_reqs = true;
+    }
+    const backendURITemplate = new Template({
+        uri: `${options.backend_host_template}/{{path}}`
+    });
+    const usePathSegment = options.use_path_segment;
+    const blockExternalReqs = options.block_external_reqs;
+    return {
+        spec: {
+            paths: {
+                '/{+path}': {
+                    'x-hidden': true,
```

Comment:
Putting this here only hides `/{+path}` but not where it's mounted, so `/v1/key_value` will still show up in the spec.

Reply:
Since the HS PR will not be merged, this needs to be moved under `all`

Reply:
Done